docs/toolkits/openstack-operations-maintenance.md

```diff
@@ -1,47 +1,181 @@
+``` bash
+openstack network list --external
+```
+Then if you have any networks listed, verify the `is_default` value when shown with:
+``` bash
+openstack network show <external network name>
+```
+If it is the default network, you can move on to the next step.  If not, set it to default with:
+``` bash
+openstack network set <external network name> --default
+```
 
+Now, we need a default subnet pool that the auto-allocated network will use. You can create this with:
+``` bash
+openstack subnet pool create --share --default --pool-prefix 192.168.0.0/16 --default-prefix-length 25 shared-default
+```
+This will create a default subnet pool using the `192.168.0.0/16` subnet, each carving off a /25 subnet off.  Each subnet will allow 128 different VMs on the network and also allow you to be able to carve off 512 subnets.  You can adjust these numbers if they won't fit your use case.  
+
+Now, your users can create an auto-allocated network with:
+``` bash
+openstack network auto allocated topology create --or-show
+```
 
-Mysql
+#### Adding and Managing Users
+
+This is probably something that is more necessary than recommended.  In this section, we will go over adding users, allocations, adjusting quotas for specific use cases, etc.  
+
+To add a user, you can run:
+``` bash
+openstack user create --domain <default domain> --password-prompt <user name>
+```
+This will ask you for a password.  You can pass this information to each user to allow them to access the OpenStack cloud.  To delete a user, it's simply:
+``` bash
+openstack user delete <user name>
+```
+
+The newly created user will need a project to operate on as well.  If they don't have a project, you can create one with:
+``` bash
+openstack project create --domain <domain name> --description "<A good description here>" <project name>
+```
+
+In OpenStack, you don't simply add a user to a project.  There needs to be a role associaction created, associating the user and the project.  The role itself can be a priviledged user or standard user.  If you don't have a standard user role created, you can create one with:
+``` bash
+openstack role create <role name>
+```
+Role associations are created with:
+``` bash
+openstack role add --project <project name> --user <user name> <role name>
+```
+
+To remove the role association, run:
+``` bash
+openstack role remove --user <user name> --project <project name> <role name>
+```
 
-Checks for a single node database:
+Oftentimes, different projects or users will have different needs of the OpenStack cloud.  This can lead to the default quotas not allowing them to do something that fits their workflow.  For example, perhaps they need more than the default quota of volumes.  These can be adjusted with the CLI by running:
+``` bash
+openstack quota set --volumes <new volume quota> <project name>
+```
+Note that quotas are set per project and not per user.
+
+
+
+### Maintenance
+
+---
+
+#### NTP
+	- What NTP Errors look like
```

Comment:
When you start with 4-space-indented bullets, Markdown assumes you want a code block, so mkdocs renders it like this:

![image](https://user-images.githubusercontent.com/13605043/187511340-9b2f8c82-f446-4ebe-9045-f7d8b964fc38.png)

You get fixed-width font, horizontal scrolling, and random bold keywords -- not great. The fix is just to start your outer-most level of bullets at the beginning of the line, and for sub-levels, indent by 2 or 4 spaces from there.

If you render with `mkdocs serve` you'll see several sections that need this tweak.

Reply:
Yikes.  I think I want to remove this format as a whole, and use something more outline-esque.